src/tests/client.test.js

```diff
@@ -0,0 +1,54 @@
+import jest from 'jest-mock';
+
+import { make_client } from '../client.js';
+
+/**
+ * Test the client
+ */
+
+describe('client', () => {
+
+  it('should return results: mock', async () => {
+
+    // keep a copy of fetch
+    const nodeFetch = fetch;
+
+    const mockResponse = {
+      body: {
+        results: [{
+          hits: [{
+            description: 'test',
+            id: 1,
+            name: 'test',
+            postgres_searchbox_v1_doc: "'coalesc':1,3 'descript':4 'name':2",
```

Comment:
Oh, I guess this comes from `SELECT *` in `handlerNextJS`. :( We should drop it, it's not useful to the client and just slows everything down.

Reply:
Sure, of course... I was pretty blind to it earlier - good spot.

I'll remove and also investigate... instantsearch may have a property in the request to affect the response columns.